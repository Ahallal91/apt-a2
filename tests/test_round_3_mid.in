BRLURBBBBRYUYBLLBBRYRBLYLUURBUUYBYRRRUBUYUBBBRUUYBRULRYUBURRYUYYRUBLYUUYLLRLYRRLYYYLLYLLLLLBULBURLRY
Test Player 1
Test Player 2
turn 5 B 2
turn 2 B 3
turn 4 L 3
turn 0 R 2
turn 1 L 3
turn 0 Y 4
turn 0 B 4
turn 3 Y 5
turn 0 B 1
turn 0 U 1
turn 0 R 5
turn 2 U 2
turn 3 U 3
turn 0 Y 4
turn 5 B 4
turn 0 L 1
turn 0 U 3
turn 4 R 3
turn 0 B 4
turn 0 Y 4
turn 0 R 5
turn 1 R 3
turn 0 Y 1
turn 0 L 6
turn 0 B 6
turn 2 U 3
turn 5 R 5
turn 3 Y 5
turn 0 B 3
turn 0 U 3
