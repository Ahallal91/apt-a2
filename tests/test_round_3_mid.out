=== Round 3 ===

Factories:
0: R R 
1: Y U B B 
2: 
3: 
4: L R Y U 
5: 

Score for Test Player 1: 8
Mosaic for Test Player 1:
1:         . || B Y . . .
2:       . . || . B . . .
3:     B B B || U L . . .
4:   . . . . || . . . B .
5: R R R R R || . . . . .
6: broken | R F

Score for Test Player 2: 13
Mosaic for Test Player 2:
1:         . || . . . U L
2:       . . || . . . R U
3:     U U U || . . B . R
4:   . . . . || . . . . Y
5: . . . Y Y || . . . . .
6: broken | U

